mobile/src/components/app/app.tsx

```diff
@@ -7,6 +7,7 @@ import 'fast-text-encoding';
 import { Root as RootNavigation } from '~/navigation/root/root.navigation';
 import { linking } from '~/config/config';
 import { styles } from './styles';
+import Toast from 'react-native-toast-message';
```

Comment:
```suggestion
import Toast from '~/component/common/common';
```
create a separate component for it